core/templates/dev/head/css/oppia.css

```diff
@@ -3928,10 +3928,17 @@ md-card.preview-conversation-skin-supplemental-card {
 }
 
 .oppia-activity-summary-tile .objective {
-  height: 60px;
+  height: 70px;
+  line-height: 1.26;
   font-family: "Rubik", "Roboto", Arial, sans-serif;
   font-size: 0.85em;
   padding: 12px;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  display: -webkit-box;
+  -webkit-box-orient: vertical;
+  -webkit-line-clamp: 3; /* Truncating to 3 lines*/
+  max-height: 4.41176471em; /*1.26 (line-height) * 3 / 0.85 (font-size) */
```

Comment:
Make the comment into a line comment -- put it the line above, and be a bit less cryptic. E.g. "This is calculated as 1.26 (line-height) * 3 lines / 0.85 (font-size)."

Also I think 4.41em probably suffices. The additional decimal places seem like spurious accuracy.